### PR TITLE
Allow automatic display derivation for Vec-based types

### LINKED ISSUE #11: Allow automatic display derivation for Vec-based types
Do something like

```rust
#[derive(Display)]
#[display("[{vec}]")]
pub struct Data {
   #[display(separator=",")]
   pub vec: Vec<String>,
}
```

### PR DESCRIPTION
**Pull request by 6293**
Closes https://github.com/rust-amplify/amplify-derive/issues/11

### CODE REVIEW

File: src/display.rs [hunk 1]
```diff
@@ -527,6 +530,35 @@ fn inner_struct(input: &DeriveInput, data: &DataStruct) -> Result<TokenStream2>
     })
 }
 
+fn format_field(field: &syn::Field, str_fmt: &str) -> Result<Option<TokenStream2>> {
+    if !has_formatters(field.ident.as_ref().unwrap(), str_fmt) {
+        return Ok(None);
+    }
+    let ident = field.ident.as_ref().unwrap();
```
Comment (dr-orlovsky) [inline src/display.rs]: nit:
```suggestion
    let ident = field.ident.as_ref().unwrap();
    if !has_formatters(ident, str_fmt) {
        return Ok(None);
    }
```

**Review by dr-orlovsky — APPROVED** [review]
ACK fa765f098cb431de9f2d03a7962dc29e7284259d

Very cool, thank you!

One small nit, but I can merge it as it is

File: src/display.rs (hunk 1 above)
Comment (6293) [reply src/display.rs]: ty

**Review by dr-orlovsky — APPROVED** [review]
ACK https://github.com/rust-amplify/amplify-derive/pull/39/commits/7f530b830995b0fc97c73741d69c69323b11562e

**Pull request merged**